src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+import { Component } from "react";
+import Navbar from "./Navbar";
+
+
+
+
+class Home extends Component{
+    render(){
+        return(
+            <div id="homepage-container">
+                <nav>
+                    <Navbar />
+                </nav>
+                <main>
+                    <h1>
+                        Welcome to GhibliApp
+                    </h1>
+                </main>
+               
+  
+            </div>
+      );
```

Comment:
```suggestion
        return(
            <div id="homepage-container">
                    <h1>
                        Welcome to GhibliApp
                    </h1>
            </div>
      );
```
You don't need to render the `Navbar` component here. Just render the `Navbar` component once inside of App.js above all of your Routes :)